stdlib/date/date.go

```diff
@@ -287,13 +287,28 @@ func init() {
 					return nil, errors.New(codes.Invalid, "missing argument unit")
 				}
 
-				if v.Type().Nature() == semantic.Time && u.Type().Nature() == semantic.Duration {
-					w, err := execute.NewWindow(u.Duration(), u.Duration(), execute.Duration{})
-					if err != nil {
-						return nil, err
+				//print(v.Type())
+				if v.Type().Nature() == semantic.Time || v.Type().Nature() == semantic.Duration && u.Type().Nature() == semantic.Duration {
```

Comment:
It might be good if you add this kind of logic to the `values` package. It's a bit difficult to read here. Something like:

```go
package values

func IsTimeable(v Value) bool {
    return v.Type().Nature() == semantic.Time || v.Type().Nature() == semantic.Duration
}
```

I mostly recommend this because I had a difficult time reading this logical condition. It occurred to me that this is testing that the value is a time or duration and the unit is a duration.